.github/workflows/pypi-publish.yml

```diff
@@ -66,14 +62,13 @@ jobs:
           
 
       - name: Run example
-        if: success()
         run: |
           poetry install
           PVAULT_SERVICE_LICENSE=${{ secrets.PVAULT_SERVICE_LICENSE }} /bin/bash run.sh
         working-directory: examples/django-encryption-example
 
       - name: Build and publish
-        if: success() && inputs.skip_publish != 'true'
+        if: inputs.bump-package != 'none'
```

Comment:
Why did you remove `success()`

Reply:
Because this is the default behavior.
Jobs stops on the first failed step unless mentioned otherwise